### ISSUE TITLE
repoze.workflow.zcml / kotti:workflow.zcml incompatiblity

### ISSUE THREAD

**Issue by mgrbyte** [Kotti/Kotti#461]
I've been attempting to port Kotti to use Python 3;
[repoze.workflow](https://pypi.python.org/pypi/repoze.workflow/1.0b1) adds an optional `title` attribute to the `state` tag,
which causes an exception when attempting to load the current (as of Kotti 1.2) workflow.zcml.

Effect:
```
TypeError: add_state() got multiple values for keyword argument 'title'
```

**Comment by tiberiuichim** [comment]
I have a branch here that fixes that: https://github.com/cruxlog/kotti/tree/upgrade_repozeworkflow

There's an additional issue that that branch wants to fix, that of eval'd titles.

Will do a PR when I know more of why the original code was like that.

**Comment by disko** [comment]
@mgrbyte Wooohoo that's great!

@tiberiuichim I left an inline comment on your code. Not sure about it though.

**Comment by disko** [comment]
Fixed in 1.3.0.
